fastembed/text/e5_onnx_embedding.py

```diff
@@ -25,6 +25,13 @@
             "hf": "xenova/paraphrase-multilingual-mpnet-base-v2",
         },
     },
+    {
+        "model": "intfloat/multilingual-e5-large-instruct",
```

Comment:
There are no ONNX files here? https://huggingface.co/intfloat/multilingual-e5-large-instruct/

This is most definitely wrong. You'll have to find a source which has ONNX Runtime support.

Reply:
I'm sorry for the inconvenience, but despite my efforts to convert it in a Colab notebook, I encountered multiple crashes.

Reply:
Could you please refer from where do I convert the model to onnx ? Previously I used your colab notebook which is given here https://github.com/qdrant/fastembed/blob/main/experiments/01_ONNX_Port.ipynb

Reply:
This is my notebook: https://colab.research.google.com/drive/1l2hn4e9LiHChf41r1Ku0to3U_JP6V75g#scrollTo=gV1u1VQsl_Pj

Reply:
I don't think quantization for this model is trivial. Even with a High RAM, no error export — this fails on the allclose

https://colab.research.google.com/drive/1XsDpneTngLdM58B6iPUIH5T2nvTCmbLL?usp=sharing

Reply:
Yeah I could see the response on the allclose like this :`Outputs are close: False
First few elements of ONNX output: [ 0.33449987  0.49482328  0.2530444  -0.91957766  0.9442016 ]
First few elements of Hugging Face output: [ 0.01167124  0.01642629  0.01465717 -0.03468534  0.03800855]`. I tried to compare the outputs of both the models but as we could see from the outputs , it failed to mach the speed. https://colab.research.google.com/drive/1XsDpneTngLdM58B6iPUIH5T2nvTCmbLL?usp=sharing#scrollTo=snCRs-POk88s

Reply:
But I have also tried with  multilingual-e5-large  and the response remain the same . here's that [notebook(https://colab.research.google.com/drive/1Dg_DmIrLUHkMh1LJ5YKfPC-jUv46qiFd)

Reply:
Could you please direct me to the essential steps for the  Quantization of this model?

Reply:
I have also tried to convert dangvantuan/sentence-camembert-base for creating embedding in French but as prev were not able to find the way to properly quantise the model , this one also stay behind that process. Here's that : https://colab.research.google.com/drive/1WnpzMumZ8cuwz9_EGFwsQnHHVjzSFre8#scrollTo=XwwmXiF7mbhx

Reply:
while optimising it, `Optimizing model...
WARNING:onnx_model:Failed to run symbolic shape inference. Please file an issue in https://github.com/microsoft/onnxruntime.` Is this the main cause for the failure of quantisation ?

Reply:
I have added the onnx file source url for now: https://drive.google.com/file/d/1XSl4RantOvSTiZeta7tl8S6f9P6MMvSo/view